barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_verifier.cpp

```diff
@@ -9,9 +9,10 @@ void ProtoGalaxyVerifier_<VerifierInstances>::receive_and_finalise_instance(cons
 {
     auto& key = inst->verification_key;
     OinkVerifier<Flavor> oink_verifier{ key, transcript, domain_separator + '_' };
-    auto [relation_parameters, witness_commitments] = oink_verifier.verify();
-    inst->relation_parameters = relation_parameters;
-    inst->witness_commitments = witness_commitments;
+    auto [relation_parameters, witness_commitments, public_inputs] = oink_verifier.verify();
+    inst->relation_parameters = std::move(relation_parameters);
+    inst->witness_commitments = std::move(witness_commitments);
+    inst->public_inputs = std::move(public_inputs);
```

Comment:
not sure if these std::move()s are doing anything here

Reply:
think it's good practice since we dont need the oink_verifier from here onwards and it makes transfer of ownership explicit